projects/m4/012-repeated-words/js/index.js

```diff
@@ -0,0 +1,88 @@
+const fs = require("fs");
+const fsPromises = fs.promises;
+const path = require("path");
+const input = path.join(__dirname, "input.txt");
+
+// MAIN FUNCTION
+function findRepeatedWords(inputFile) {
+  fsPromises.readFile(inputFile, "utf8").then((data) => {
+    const lines = data.split("\n");
+    let tempWord = "";
+    let lineNumber = 1;
+
+    lines.forEach((line) => {
+      const arrayLine = getArray(line);
+      arrayLine.forEach((word) => {
+        if (word === tempWord) {
+          console.log(
```

Comment:
It might be a good idea to create an **object** with a **word** and a **line number**. 
Then print everything at the end, outside the function _(you can also use_ `async/await` _if needed)_